Welthungerhilfe/cgm-ml
# Build cgm-ml-common sdist PyPI

**Motivation**: For databricks and potentially more use cases, we need to use the cgm-ml common code from outside of this repository. 

**Solution**: When releasing, we also publish the release to PyPI. 

**How to check**
- Install the cgm-ml-common dependency: `pip install cgm-ml-common`
- Start a python REPL: `python`
- Run an import command in the REPL: `from src.common.model_utils.lr_finder import LRFinder` should work

**Future work:** 
- adjust version: currently `3.0.0a4` but this should probably be the same as the GitHub release number
- rename `src` to `cgmml`: Let's do this later because this is a bigger change
- add other authors to setup.py
- Refactor: zscore package could use github actions instead of azure pipeline
